lib/client/jsx/actions/manifest_actions.js

```diff
@@ -71,45 +69,44 @@ export const requestManifests = ()=>{
       showErrors(err, dispatch);
     };
 
-    return ManifestAPI.fetchManifests(new Exchange(dispatch,'request-manifest'))
+    return fetchManifests(new Exchange(dispatch,'request-manifest'))
       .then(localSuccess)
       .catch(localError);
   };
 };
 
 // Delete a manifest from the database and the store.
-export const deleteManifest = (manifest_id)=>{
+export const deleteManifest = (manifest, success)=>{
```

Comment:
This and the previous line were changed to bring the manifest selection into the state. Specifically, it does not make sense to me that whenever I fire the action 'deleteManifest', with the intent of deleting a manifest, I also get a selectManifest(null) action which changes the UI state. The dependency between the deleteManifest action and the selectManifest() action is an implementation choice and shouldn't be coupled in the action.

So, I moved it into a success handler, which gets called in manifests.jsx:

    this.props.deleteManifest(manifest, () => this.selectManifest(0));                                                                                                                       

I also opted for having selectManifest set in the top-level component state, rather than in the store, since multiple top-level components might specify different manifests and the intersections might be confusing.